lib/rollbar/delay/sidekiq.rb

```diff
@@ -10,7 +10,7 @@ def initialize(*args)
       end
 
       def call(payload)
-        if (::Sidekiq::Client.push @options.merge('args' => [payload]) == nil)
+        if ::Sidekiq::Client.push(@options.merge('args' => [payload])) == nil
```

Comment:
Previous code was sending the result of the `==` expression to `#push` instead of the options hash.

Reply:
Good catch, this is crazy, I was suddenly worried that this has somehow been here for a long time but just saw that the conditional is new enough not to have been released yet.

Reply:
@waltjones thanks for catching that